### DISCUSSION TITLE
Trivy http proxy from secret

### DISCUSSION THREAD

**Discussion by kimdre** [aquasecurity/trivy-operator#2171] [Help & Support] [answered]
Hi, the helm chart provides a key to set values for the operator from a secret which I need because my http proxy config is inside a secret. 

```yaml
operator:
  valuesFromSecret: proxy-secrets
```

Is there a option to do the same to set the http proxy settings for trivy?

```y
trivy.httpProxy
trivy.httpsProxy
```

https://github.com/aquasecurity/trivy-operator/blob/7b0c518a622febde21e97c9dd5ac6c9b27ea60ed/deploy/helm/values.yaml#L384-L391

**Comment by chen-keinan** [comment]
sure, can be done, please open an issue for it

**Reply by kimdre** [reply]
Ok, I opened https://github.com/aquasecurity/trivy-operator/issues/2172

**Reply by chen-keinan** [reply]
> Ok, I opened #2172

great, closing the discussion

**Comment by kimdre** [comment, marked as answer]
Proxy config from a secret is specified like this:

Specify the secret resource with

https://github.com/aquasecurity/trivy-operator/blob/7b0c518a622febde21e97c9dd5ac6c9b27ea60ed/deploy/helm/values.yaml#L184-L185

The secret resource should look like this, values are the same as you would specify on linux environments.

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: proxy-secrets
data:
  HTTP_PROXY: ...
  HTTPS_PROXY: ...
  NO_PROXY: ...
```
